src/utils/calcModularScale.js

```diff
@@ -10,8 +10,10 @@ module.exports = (value, data) => {
         const sFtSMax = data.settings?.screenMax
         const unit = data.unit
         const sFtUnit = typeof unit === 'string' ? unit : 'rem';
-        const ftMin = sFtMin * Math.pow(sFtRMin, value);
-        const ftMax = sFtMax * Math.pow(sFtRMax, value);
+        const minValue = sFtMin * Math.pow(sFtRMin, value);
+        const maxValue = sFtMax * Math.pow(sFtRMax, value);
+        const ftMin = Math.min(minValue, maxValue);
```

Comment:
@salmans-teikametrics 
Sorry for being a bit late. I've looked into it now and it makes sense what you've said. But I prefer to handle it a bit differently. 

Can you change this line into this: `const ftMin = minValue;`?
I would say that's a more robust / expected behavior.

Then I'll merge your PR. 

Thanks for your effort!